src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -457,11 +465,54 @@ static Status deserializeTensor(const std::string& requestedName, const KFSReque
     return StatusCode::OK;
 }
 
+typedef std::unique_ptr<OvmsPyTensor> OvmsPyTensorPtr;
+
+#if (PYTHON_DISABLE == 0)
+static Status deserializeTensor(const std::string& requestedName, const KFSRequest& request, std::unique_ptr<PyObjectWrapper<py::object>>& outTensor, PythonBackend * pythonBackend) {
```

Comment:
Deserialization has changed alot in this release due to streaming, please check latest commits

Reply:
Already rebased. I'll check manually and run tests to make sure it's all good.